.github/workflows/ci.yml

```diff
@@ -0,0 +1,169 @@
+name: CI
+
+on: push
+
+env:
+  # Would be more usual to set R_LIBS_USER, but R uses R_LIBS first if present
+  # ...and it's always present here, because the rocker/tidyverse base image
+  # checks at R startup time for R_LIBS and R_LIBS_USER, sets both if not found
+  R_LIBS: ~/R/library
+  _R_CHECK_LENGTH_1_CONDITION_: true
+  _R_CHECK_LENGTH_1_LOGIC2_: true
+  # Avoid compilation check warnings that come from the system Makevars, not us
+  # See https://stat.ethz.ch/pipermail/r-package-devel/2019q2/003898.html
+  _R_CHECK_COMPILATION_FLAGS_KNOWN_: -Wformat -Werror=format-security -Wdate-time
```

Comment:
These flags only affect the operation of R CMD check. Might make sense to set them inside the `check` job rather than for the entire workflow?